src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -1,958 +1,217 @@
-
-                            /* needed because when the tab contains HTML contents, user can click */
-                            /* on any of those elements instead of their parent '<button>' element */
-                            while (activeTab.tagName.toLowerCase() !== 'button') {
-                                activeTab = activeTab.parentNode;
-                            }
-
-                            /* get the full list of tabs through the parent of the active tab element */
-                            var tabNavigation = activeTab.parentNode.children;
-                            for (var k = 0; k < tabNavigation.length; k++) {
-                                var tabId = tabNavigation[k].getAttribute('data-tab-id');
-                                document.getElementById(tabId).className = 'hidden';
-                                removeClass(tabNavigation[k], 'active');
-                                tabNavigation[k].removeAttribute('aria-selected');
-                                tabNavigation[k].setAttribute('tabindex', '-1');
-                            }
-
-                            addClass(activeTab, 'active');
-                            activeTab.setAttribute('aria-selected', 'true');
-                            activeTab.removeAttribute('tabindex');
-                            var activeTabId = activeTab.getAttribute('data-tab-id');
-                            document.getElementById(activeTabId).className = 'block';
-                        });
+                    if ('' !== window.getSelection().toString()) {
+                        /* Don't do anything on text selection */
+                        return;
                     }
 
-                    tabGroups[i].setAttribute('data-processed', 'true');
-                }
-            },
-
-            createToggles: function() {
-                var toggles = document.querySelectorAll('.sf-toggle:not([data-processed=true])');
+                    let toggle = e.target || e.srcElement;
 
-                for (var i = 0; i < toggles.length; i++) {
-                    var elementSelector = toggles[i].getAttribute('data-toggle-selector');
-                    var element = document.querySelector(elementSelector);
-
-                    addClass(element, 'sf-toggle-content');
-
-                    if (toggles[i].hasAttribute('data-toggle-initial') && toggles[i].getAttribute('data-toggle-initial') == 'display') {
-                        addClass(toggles[i], 'sf-toggle-on');
-                        addClass(element, 'sf-toggle-visible');
-                    } else {
-                        addClass(toggles[i], 'sf-toggle-off');
-                        addClass(element, 'sf-toggle-hidden');
+                    /* needed because when the toggle contains HTML contents, user can click */
+                    /* on any of those elements instead of their parent '.sf-toggle' element */
+                    while (!toggle.classList.contains('sf-toggle')) {
```

Comment:
you could use `toggle.closest('.sf-toggle')` instead of a manual implementation of it.